src/bindings/python/src/pyopenvino/graph/node_output.cpp

```diff
@@ -13,3 +13,31 @@ namespace py = pybind11;
 
 template void regclass_graph_Output<ov::Node>(py::module m, std::string typestring);
 template void regclass_graph_Output<const ov::Node>(py::module m, std::string typestring);
+
+template <typename T>
+void def_type_dependent_functions(py::class_<ov::Output<T>, std::shared_ptr<ov::Output<T>>>& output) {}
+
+template <>
+void def_type_dependent_functions<ov::Node>(
+    py::class_<ov::Output<ov::Node>, std::shared_ptr<ov::Output<ov::Node>>>& output) {
+    output.def("remove_target_input",
+               &ov::Output<ov::Node>::remove_target_input,
+               py::arg("target_input"),
+               R"(
+            Removes a target input from the output referenced by this output handle.
+
+            Returns
+            ----------
+            None
+        )");
+    output.def("replace",
+               &ov::Output<ov::Node>::replace,
+               py::arg("replacement"),
+               R"(
+            Replace all users of this value with replacement.
+
+            Returns
+            ----------
+            None
+        )");
```

Comment:
Update docs to new style. Here `None` means the same as void, there is no need to document default returning value.